lib/membrane_vpx/decoder/vp9_decoder.ex

```diff
@@ -0,0 +1,51 @@
+defmodule Membrane.VP9.Decoder do
+  @moduledoc """
+  Element that decodes a VP9 stream
+  """
+  use Membrane.Filter
+
+  alias Membrane.{VP9, VPx}
+
+  def_options width: [
+                spec: non_neg_integer() | nil,
+                default: nil,
+                description: """
+                Width of a frame, assumed if not provided with stream format. In case it's not specified in either case the element will crash.
```

Comment:
```suggestion
                Width of a frame, needed if not provided with stream format. In case it's not specified in either case the element will crash.
```